src/box/box.cc

```diff
@@ -2558,8 +2567,30 @@ struct sync_trigger_data {
 	int count;
 	/** Whether the request is timed out. */
 	bool is_timed_out;
+	/** Count of fibers that are using data. */
+	int ref_count;
 };
 
+/** Let others know we need data. */
+void
+sync_trigger_data_ref(struct sync_trigger_data *data)
+{
+	++data->ref_count;
+}
+
+/*
+ * Let others know that we no longer need the data.
```

Comment:
Nit: comments describing structure members and function declarations should start with `/**`.